config/data_sources/AGG_cdc_restricted.sql

```diff
@@ -14,38 +14,47 @@ WITH
       SELECT DISTINCT
         b.state_fips_code as state_fips,
         b.state_name,
-        a.race_and_ethnicity,
+        a.race_category_id,
         a.cases, a.hosp_y, a.hosp_n, a.hosp_unknown, a.death_y, a.death_n, a.death_unknown
     FROM `cdc_restricted_data.cdc_restricted_by_race_state` AS a
     LEFT JOIN `bigquery-public-data.census_utility.fips_codes_states` AS b
         ON a.state_postal = b.state_postal_abbreviation
     WHERE a.state_postal != "Unknown"
   ),
   joined_with_acs as (
-      SELECT x.*, y.population
+      SELECT x.*, y.race, y.race_includes_hispanic, y.race_and_ethnicity, y.population
       FROM cdc_restricted_race_state AS x
       LEFT JOIN `acs_population.by_race_state_std` AS y
-          USING (state_fips, race_and_ethnicity)
+          USING (state_fips, race_category_id)
   ),
   total_rows as (
       SELECT
         state_fips, state_name,
-        'Total' as race_and_ethnicity,
+        'TOTAL' as race_category_id,
         SUM(cases) as cases,
         SUM(hosp_y) as hosp_y,
         SUM(hosp_n) as hosp_n,
         SUM(hosp_unknown) as hosp_unknown,
         SUM(death_y) as death_y,
         SUM(death_n) as death_n,
         SUM(death_unknown) as death_unknown,
+        'Total' as race,
+        CAST(NULL as BOOL) as race_includes_hispanic,
+        'Total' as race_and_ethnicity,
```

Comment:
I thought we had decided to use the race_and_ethnicity after all based on feedback from MSM? Also I remember discussing that it would be a decently large refactor to make the frontend always use "race" instead of "race_and_ethnicity", and we didn't think it was necessarily worth doing all at once? I thought we would just supply both and the frontend can use whichever it needs. The only downside to doing both is data transfer size/memory usage.

Reply:
Right yeah, I was referring to the larger refactor on the FE to use race instead of race_and_ethnicity, which we can do over time. I think we're on the same page